app/src/main/java/com/example/vybe/MyVibesActivity.java

```diff
@@ -90,6 +91,8 @@ protected void onCreate(Bundle savedInstanceState) {
         socialBtn = findViewById(R.id.social_btn);
         profileBtn = findViewById(R.id.profile_btn);
 
+        vibeEventDBPath = "Users/" + mAuth.getCurrentUser().getUid() + "/VibeEvents";
+        Log.d(TAG, "onCreate: " + vibeEventDBPath);
```

Comment:
Do we want to keep this log for now for debugging?

Reply:
I can remove the log. It was more for my testing when I was doing the work

Reply:
Removed